ContentMetadata/ContentMetadata.cs

```diff
@@ -2,63 +2,69 @@
 
 namespace ContentMetadata
 {
-    [Serializable]
-    public class ContentMetadata
-    {
-        public ContentMetadata() { }
+	[Serializable]
+	public class ContentMetadata
+	{
+		public ContentMetadata() { }
 
-        public ContentMetadata(Guid contentId, Guid contentTypeId, string key, string value)
-        {
-            ContentId = contentId;
-            ContentTypeId = contentTypeId;
-            Key = key;
-            Value = value;
-        }
+		public ContentMetadata(Guid contentId, Guid contentTypeId, string key, string value, int userId = 0)
```

Comment:
This is still referencing client specific use cases e.g. the UserId. Is the UserId needed here? The ContentId for the user should be sufficient for a lookup.